exercises/isogram/README.md

```diff
@@ -0,0 +1,22 @@
+# Isogram
+
+Determine if a word or phrase is an isogram.
+
+An isogram (also known as a "nonpattern word") is a word or phrase without a repeating letter, however spaces and hyphens are allowed to appear multiple times.
+
+Examples of isograms:
+
+- lumberjacks
+- background
+- downstream
+- six-year-old
+
+The word *isograms*, however, is not an isogram, because the s repeats.
+
+If everything goes well, you will be ready to fetch your first real exercise.
```

Comment:
Is this a copy from elsewhere?

Reply:
from [https://github.com/exercism/problem-specifications/blob/master/exercises/isogram/description.md](https://github.com/exercism/problem-specifications/blob/master/exercises/isogram/description.md)

Reply:
Ohh, probably from hello world, I Should remove it